rotkehlchen/chain/ethereum/modules/uniswap/v2/common.py

```diff
@@ -32,14 +32,14 @@
 
 def decode_uniswap_v2_like_swap(
         tx_log: EvmTxReceiptLog,
-        decoded_events: list[HistoryBaseEntry],
+        decoded_events: list['EvmEvent'],
         transaction: EvmTransaction,
         counterparty: str,
         database: 'DBHandler',
         ethereum_inquirer: 'EthereumInquirer',
-        notify_user: Callable[[HistoryBaseEntry, str], None],
-) -> tuple[Optional[HistoryBaseEntry], list[ActionItem]]:
-    """Common logic for decoding uniswap v2 like protocols (uniswap and sushiswap atm)
+        notify_user: Callable[['EvmEvent', str], None],
+) -> tuple[Optional['EvmEvent'], list[ActionItem]]:
+    """Common logic for decoding uniswap v2 like counterpartys (uniswap and sushiswap atm)
```

Comment:
Leave it as it was. Even if the variable ends up staying coutnerparty this is for protocols.

```suggestion
    """Common logic for decoding uniswap v2 like protocols(uniswap and sushiswap atm)
```